include/bdsm.h

```diff
@@ -32,8 +32,8 @@
 #define __BDSM_H_
 
 #define BDSM_VERSION_CURRENT  4
```

Comment:
Oh wait this isn't libtool anymore, please ignore

Reply:
This is anyway just library version which does not necessarily matches so versions due to the way libtool calculated that based on the "age" versioning thing. However with the transition from libtool to meson we need to make sure we end up with at least the same or higher so version as before else this might cause some trouble…